common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -4,36 +4,35 @@
 @import conf.switches.Switches.SearchSwitch
 
 @sectionList(topLevelSection: NewNavigation.EditionalisedNavigationSection) = {
-    <li class="menu-item js-navigation-item"
-        data-section-name="@topLevelSection.name">
-        <details class="js-close-nav-list"
-                 id="primary-list-@topLevelSection.name"
-                 open>
-            <summary class="menu-item__title js-navigation-button"
-                     data-link-name="nav2 : secondary : @topLevelSection.name">
+    @defining(Edition(request)) { edition =>
+        <li class="menu-item js-navigation-item"
+            data-section-name="@topLevelSection.name">
+            <a href="@{topLevelSection.getPopularEditionalisedNavLinks(edition)(0).url}"
```

Comment:
I think it would be better to add another `def` in this trait for `url` or `path` https://github.com/guardian/frontend/blob/master/common/app/common/NewNavigation.scala#L26

Or what might be better is to have a helper function which looks up the correct url per TopLevelSection by edition. I can come over and sit with you today if it helps :)